code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1142,6 +1142,18 @@
 	reagents.add_reagent(CARAMEL, 4)
 	bitesize = 2
 
+/obj/item/weapon/reagent_containers/food/snacks/bearburger
+	name = "bear burger"
+	desc = "There's gotta be a pun in there somewhere, but no Pun Pun."//TODO: replace with a better pun, bear-related if possible
```

Comment:
Bearly good!

Reply:
It's unbearable how good it is.

Reply:
"It's unbearably grizz(t)ly"

Reply:
good to crack open a bear with

Reply:
Fits perfectly in any pic-a-nic basket. Oh bothering to grizzle into this won't be a boo-boo. Honey, it would be beary foolish to hibernate on such a unbearably, ursa majorly good treat!